### PR TITLE
add n2log-unscramble

### PR DESCRIPTION
**Pull request by pgerber**

### CODE REVIEW

File: bin/n2log-unscramble [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        print()
+        for key, value in mdc:
+            print('{} = {}'.format(key, value), file=output)
+
+
+def is_level_enabled(level, level_enabled):
+    if level is None:
+        return True
+
+    if LEVELS[level_enabled] <= LEVELS[level.lower()]:
```
Comment (ottrob) [inline bin/n2log-unscramble:43]: `return LEVELS[level_enabled] <= LEVELS[level.lower()]` ?

File: bin/n2log-unscramble [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        try:
+            msg = json.loads(line)
+
+            if not is_level_enabled(msg.get('level'), level):
+                continue
+
+            for value in msg.values():
+                if predicate(str(value)):
+                    break
+            else:
```
Comment (ottrob) [inline bin/n2log-unscramble:59]: wrong line indent on line 59. `else` belongs to `if` on line 57 i guess

Comment (pgerber) [reply bin/n2log-unscramble:59]: No, that really does belong to the `for`.

https://docs.python.org/3/reference/compound_stmts.html#for

**Comment by pgerber** [comment]
Sorry, accidentally pushed to master. Time to require a review and and protect the master branch.

**Pull request merged**